include/erlcloud_aws.hrl

```diff
@@ -7,6 +7,8 @@
           s3_host="s3.amazonaws.com"::string(),
           s3_port=80::non_neg_integer(),
           s3_follow_redirect=false::boolean(),
+          s3_follow_redirect_count=2::non_neg_integer(),
+          s3_bucket_access_method=vhost::vhost|path,
```

Comment:
I think there is no reason to duplicate s3 API. Bucket access method only affects the way a request URL is constructed - either https://bucket-name.s3.amazonaws.com or https://s3.amazonaws.com/bucket-name/